judgels-frontends/raphael/src/routes/contests/contests/single/components/ContestAnnouncementsWidget/ContestAnnouncementsWidget.jsx

```diff
@@ -1,6 +1,7 @@
 import { Tag } from '@blueprintjs/core';
 import * as React from 'react';
 import { connect } from 'react-redux';
+import { selectContest } from '../../../modules/contestSelectors';
```

Comment:
Please group this with other `{ select*** }` imports

Reply:
I'm not sure what you meant, do you mean something like this? 

![image](https://user-images.githubusercontent.com/31788267/101770391-5ae84e00-3b23-11eb-84be-93e6b28c7dff.png)

Reply:
Yes

Reply:
Sorry2 what I meant was

```
import { connect } from 'react-redux';

import { selectContest } from '../../../modules/contestSelectors';
import { selectContestWebConfig } from '../../../modules/contestWebConfigSelectors';
```